src/app/views/project/config/project-config.component.js

```diff
@@ -32,7 +32,7 @@ class ConfigController {
     }
   }
 
-  updateStateData() {
+  updateShowSave() {
     this.showSave = !angular.equals(this.currentProject.config, this.config);
     if (this.showSave) {
```

Comment:
this if/else has no use now

Reply:
either does `projectConfigState` and `this.$state`. please remove all references